src/pages/patientView/PatientViewPage.tsx

```diff
@@ -466,8 +477,25 @@ export default class PatientViewPage extends React.Component<
 
     @autobind
     onMutationTableRowClick(d: Mutation[]) {
-        if (d.length) {
-            this.patientViewMutationDataStore.toggleSelectedMutation(d[0]);
+        // if one mutation, select it and toggle off other selected
+        if (d.length == 1) {
+            // this.patientViewMutationDataStore.toggleSelectedMutation(d[0]);
+            this.patientViewMutationDataStore.setSelectedMutations(d);
+            if (this.patientViewCnaDataStore.selectedCna[0]) {
+                this.patientViewCnaDataStore.toggleSelectedCna(
+                    this.patientViewCnaDataStore.selectedCna[0]
+                );
+            }
+            this.handleLocusChange(d[0].gene.hugoGeneSymbol);
```

Comment:
this `if` block is almost identical to the `else if` block below. It may worth trying to always do `this.patientViewMutationDataStore.setSelectedMutations([d[0]])` and get rid of the conditional selection. Because you are always selecting the first mutation of the array no matter what, right?